src/dispatching.rs

```diff
@@ -187,12 +187,36 @@
 //! useful features. See [`examples/dispatching_features.rs`] as a more involved
 //! example.
 //!
+//! ## Dispatching or REPLs?
+//!
+//! The difference between dispatching and the REPLs ([`crate::repl`] & co) is
+//! that dispatching allows you a greater degree of flexibility at the cost of a
+//! bit more complicated setup.
+//!
+//! Here are things that dispatching can do, but REPLs can't:
+//!  - Handle different kinds of [`Update`];
+//!  - [Pass dependencies](struct.DispatcherBuilder.html#method.dependencies) to
+//!    handlers;
+//!  - Disable a [default Ctrl-C
+//!    handling](struct.DispatcherBuilder.html#method.enable_ctrlc_handler);
+//!  - Control your
+//!    [default](struct.DispatcherBuilder.html#method.default_handler) and
+//!    [error](struct.DispatcherBuilder.html#method.error_handler) handlers;
+//!  - Use [dialogues](dialogue/index.html).
```

Comment:
`Pass dependencies`, `default Ctrl-C handling`, `default`, `error`, `dialogues` links should not be defined inline.

Also I think you can use paths (aka intra doc links) instead of `.html` urls

Reply:
I tried using paths but it didn't worked.

Reply:
that's weird 🤔

Reply:
Will you give it a try or we'll leave the paths as-is? (I'll also fine to make the paths non-inline.)

Reply:
I'll give it a try

Reply:
Done in 1942b8f49b7987e205cc8444b0a4d890eacc049e (don't forget to pull it, if you'll make further changes)